pegasus/sites.v3/code.org/public/educate/it.md

```diff
@@ -27,70 +27,71 @@ Starting in July 2024, we will update the minimum versions required for browsers
           Browser
         </th>
         <tr>
-          <td>Chrome 96.x</td>
+          <td>Chrome 103.x</td>
         </tr>
         <tr>
-          <td>Safari 13.x</td>
+          <td>Safari 14.x</td>
         </tr>
         <tr>
-          <td>Mobile Safari 13.x</td>
+          <td>Mobile Safari 14.x</td>
         </tr>
         <tr>
-          <td>Edge 94.x</td>
+          <td>Edge 119.x</td>
         </tr>
         <tr>
-          <td>Firefox 91.x</td>
+          <td>Firefox 98.x</td>
         </tr>
       </table>
     </td>
     <td style="vertical-align: top; border-color: transparent; padding: 0px 12px 0px 0px;">
       <table>
         <th>
-          Browser July '24
+          Platform
         </th>
         <tr>
-          <td>Chrome 103.x</td>
+          <td>macOS 10.15</td>
         </tr>
         <tr>
-          <td>Safari 14.x</td>
+          <td>iOS 15.x</td>
         </tr>
         <tr>
-          <td>Mobile Safari 14.x</td>
+          <td>Windows 8</td>
         </tr>
         <tr>
-          <td>Edge 119.x</td>
+          <td>Android 8.x</td>
         </tr>
         <tr>
-          <td>Firefox 98.x</td>
+          <td>ChromeOS (Chromebooks)</td>
         </tr>
       </table>
     </td>
+    <!-- "coming soon" system requirements, to be updated and made live from March through July.
```

Comment:
Note that I have moved the "Platform" table up, and commented out the coming-soon 2025 tables.